pyinstaller_script.py

```diff
@@ -70,27 +69,18 @@ def ui_to_py(dirname: str, py_exec: str):
             '"""',
         ]
         if PIXMAP_STR in inlines:
-            # TODO determine if existing sys import is sufficient
-            # outlines.append('import sys')
             for inline in inlines.split('\n'):
                 if PIXMAP_STR not in inline:
                     outlines.append(inline)
                 else:
                     first, icon_path, last = inline.split('"')
-                    indent = first.split(ICON_VAR)[0]
-                    outlines.append(
-                        f'{indent}if getattr(sys, "frozen", False):')
-                    outlines.append(
-                        f'{TAB}{first}r"{utils.utils.LocalPath("..", icon_path)}"{last}')
-                    outlines.append(
-                        f'{indent}else:')
+                    icon_path = icon_path.split("..")[1]
                     outlines.append(
-                        f'{TAB}{first}r"{utils.utils.LocalPath(".", icon_path)}"{last}')
+                        f'{first}r"..{utils.utils.LocalPath(icon_path)}"{last}')
```

Comment:
Why no more `sys.frozen` check?  And is `split("..")[1]` index always giving consistent results?

Reply:
No more `sys.frozen` becausse 'the '.py' are only loaded if the application is frozen (build) so we don't need that if statement in the ui files anymore.' As example: 
```
if getattr(sys, 'frozen', False):
    super().setupUi(self)
else:
    PyQt6.uic.loadUi("gui/ui_files/tabUpDownload.ui", self)
```

And yess, for as far as I know the  '..' operator specifies the parent folder on all operating systems, but I can currently only test on windows and ubuntu.

Reply:
> And is split("..")[1] index always giving consistent results?

What I meant about consistent was, is there only ever 1 `..` in `icon_path`?  I remember previous examples where there are multiple such strings.  If the text following the last `..` is always the one you want, this will work best:

```python
                    icon_path = icon_path.split("..")[-1]
```

That will always give the _last_ element in the split list, **and** gives the correct result even if `..` is not in `icon_path`.

Reply:
Yes, I understand. Unless we move the icons folder there will always be one and only one `..` in the path. But I agree that always getting the last element is even nicer. Updated this line.

Reply:
Thanks!